src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -330,9 +330,9 @@ impl DeviceManager {
                 // No need to do find device for hybrid vsock device.
                 Arc::new(Mutex::new(HybridVsockDevice::new(&device_id, hvconfig)))
             }
-            DeviceConfig::VsockCfg(_vconfig) => {
+            DeviceConfig::VsockCfg(vconfig) => {
                 // No need to do find device for vsock device.
-                Arc::new(Mutex::new(VsockDevice::new(device_id.clone()).await?))
+                Arc::new(Mutex::new(VsockDevice::new(device_id.clone(), vconfig).await?))
```

Comment:
```suggestion
                Arc::new(Mutex::new(
                    VsockDevice::new(device_id.clone(), vconfig).await?,
                ))
```

in order to make `cargo fmt -- --check` happy and fix https://github.com/kata-containers/kata-containers/actions/runs/7220732032/job/19674258988?pr=8672 .